Project Academy/10. Room-test/kotlin/app/src/test/java/com/dicoding/academies/ui/detail/DetailCourseViewModelTest.kt

```diff
@@ -49,4 +51,22 @@ class DetailCourseViewModelTest {
 
         verify(observer).onChanged(dummyCourseWithModule)
     }
+
+    @Test
+    fun setBookmark() {
+        val dummyCourseWithModule = Resource.success(DataDummy.generateDummyCourseWithModules(dummyCourse, false))
+        val course = MutableLiveData<Resource<CourseWithModule>>()
+        course.value = dummyCourseWithModule
+
+        `when`(academyRepository.getCourseWithModules(courseId)).thenReturn(course)
+        viewModel.courseModule.observeForever(observer)
+
+        val courseFromViewModel = viewModel.courseModule.value!!.data
```

Comment:
Ini buat apa? Skenario berikutnya `doNothing`, jadi bisa langsung pakai `dummyCourseWithModule` saja.

Reply:
Ok mas fix it